setup.cfg

```diff
@@ -20,3 +20,4 @@ ignore =
     I200,
 
 banned-modules = tensorflow = use mlagents.tf_utils instead (it handles tf2 compat).
+                 logging = use mlagents_envs.logging_util instead
```

Comment:
This prevents `logging` from being imported, so that all new code will use `logging_util .get_logger` instead.